spyderlib/plugins/configdialog.py

```diff
@@ -731,6 +731,12 @@ def setup_page(self):
                                            'windows_style',
                                            default=self.main.default_style)
 
+        themes = ['Spyder 2', 'Spyder 3']
+        icon_choices = list(zip(themes, [theme.lower() for theme in themes]))
+        icons_combo = self.create_combobox(_('Icon theme'), icon_choices,
+                                           'icon_theme', default='spyder 2',

```

Comment:
Maybe the default keyword should be removed from these `create_*` functions, as it seems that it shouldn't be used anymore ?